Configure

```diff
@@ -116,7 +116,7 @@ my $usage="Usage: Configure [no-<cipher> ...] [enable-<cipher> ...] [-Dxxx] [-lx
 # code, so we just tell compiler to be pedantic about everything
 # but 'long long' type.
 
-my $gcc_devteam_warn = "-DDEBUG_UNUSED"
+my $devteam_common_warn = "-DDEBUG_UNUSED"
         . " -Wswitch"
```

Comment:
While you're at it, could you move -Wswitch till after -Wextra below? I mean other additional -Wflag are placed below -Wextra, and so should -Wswitch.